Code/GraphMol/ForceFieldHelpers/Wrap/rdForceFields.cpp

```diff
@@ -86,6 +86,14 @@ python::object FFConfsHelper(ROMol &mol, ForceFields::PyForceField &ff,
   return pyres;
 }
 
+ForceFields::PyForceField *GetEmptyForceField(ROMol &mol) {
+  ForceFields::ForceField *ff =
+      ForceFieldsHelper::constructEmptyForceField(mol);
+  auto *res = new ForceFields::PyForceField(ff);
```

Comment:
After you've updated `constructEmptyForceField()` to return a `unique_ptr`, this will need to change to:
```suggestion
  auto ff =
      ForceFieldsHelper::constructEmptyForceField(mol);
  auto *res = new ForceFields::PyForceField(ff.release());
```